vnext/JSI.Desktop.UnitTests/RunUnitTests.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "gtest/gtest.h"
+#include "windows.h"
+
+int main(int argc, char **argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
```

Comment:
>   [](start = 0, length = 2)

(nit) use tabs in C++ code

Reply:
We use spaces everywhere else in Microsoft/react-native-windows. I think this is to conform with Facebook's styling.